sklearn/linear_model/tests/test_ransac.py

```diff
@@ -138,6 +139,18 @@ def test_ransac_predict():
     assert_equal(ransac_estimator.predict(X), np.zeros(100))
 
 
+def test_ransac_resid_thresh_no_inliers():
+    """when residual_threshold=0.0 there are no inliers and a
```

Comment:
please use comments, not docstrings on tests. That way nose prints the function name.